test/index.js

```diff
@@ -675,7 +675,7 @@ test ('equals', function() {
   eq (Z.equals (Math.sin, Math.cos), false);
   eq (Z.equals (Identity (Identity (Identity (0))), Identity (Identity (Identity (0)))), true);
   eq (Z.equals (Identity (Identity (Identity (0))), Identity (Identity (Identity (1)))), false);
-  eq (Z.equals (Useless, Useless), false);
+  eq (Z.equals (Useless, Useless), true);
```

Comment:
Take note that `Z.equals (Lazy$of (0), Lazy$of (0))` is still `false`. But now:

```js
const x = Lazy$of (0);
Z.equals (x, x)
```

should be `true`